scripts/dump_endfield_map_marks_public.py

```
@@ -76,24 +76,28 @@ def main():
+                item["id"]: item["name"].strip()
                 for item in data.get("markTemplates", [])
             }
 
-            all_names.update(template_map.values())
+            all_names.update(name for name in template_map.values() if name)
 
-            for mark in data.get("marks", []):
-                name = template_map.get(mark["templateId"])
+            def _add_point(mark: dict):
+                nonlocal duplicate_count
+                name = template_map.get(mark.get("templateId"))
```

Ruff (0.16.2)

[warning] Function definition does not bind loop variable `template_map`

(B023)